ksqldb-functional-tests/src/test/resources/historical_plans/protobuf_-_protobuf_inference_-_other_scalars/7.1.0_1635272037962/plan.json

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
Is it true that the only QTTs which needed to be updated are those that use schema inference without explicitly specifying KEY_SCHEMA_ID or VALUE_SCHEMA_ID in the WITH clause properties?

Also, is it true that the only changes to the historic plans is that KEY_SCHEMA_ID and/or VALUE_SCHEMA_ID is now being written into the plan?

(Want to check my understanding of the changes in this PR. Thanks.)